### PR TITLE
Add CORS

### PR DESCRIPTION
**Pull request by ostapneko**
Add CORS headers for

- www.theguardian.com
- m.code.dev-theguardian.com
- membership.theguardian.com

@rtyley @kenlim @jbreckmckye @davidrapson : Do you think we covered all the cases for the time being?

### CODE REVIEW

File: membership-attribute-service/conf/PROD.conf [hunk 1]
```diff
@@ -13,3 +13,12 @@ sentry.dsn="https://b4537d8d803a4f58b3d19daa71435544:7d2cbef1b828492aabe30ed494c
 
 touchpoint.backend.default=PROD
 touchpoint.backend.test=UAT
+
+play.filters.cors.allowedOrigins = [
+  "http://www.theguardian.com",
+  "https://www.theguardian.com",
+  "http://m.code.dev-theguardian.com",
+  "https://m.code.dev-theguardian.com",
+  "http://membership.theguardian.com",
```
Comment (rtyley) [inline membership-attribute-service/conf/PROD.conf]: You can remove this, we're https only!

**Comment by jbreckmckye** [comment]
So, what's going to be the best way for us to make a CORS request to this endpoint from dev instances running under localhost?

**Comment by davidrapson** [comment]
So, actually thinking about it we likely need to cover non-prod environments and include Identity:

```
// Identity local/code/prod
"profile.thegulocal.com",
"profile.code.dev-theguardian.com",
"profile.theguardian.com",

// theguardian.com local/code/preview/prod
"www.thegulocal.com",
"m.code.dev-theguardian.com",
"preview.gutools.co.uk",
"www.theguardian.com",
```

**Comment by ostapneko** [comment]
@jbreckmckye 

- if you're running the API locally, no problem (the settings apply to the PROD env)
- if you're using the prod API from a website running locally, well, I probably need to add localhost and somehow allow any port

**Comment by ostapneko** [comment]
@davidrapson Do we expect any calls from profile?

**Comment by davidrapson** [comment]
@ostapneko Yeah, I remembered we call `/user/me/details` from profile at the moment, so we'll probably want to replace is with this API.

**Comment by ostapneko** [comment]
@davidrapson all right, all right ...

**Comment by ostapneko** [comment]
@jbreckmckye It looks like there is no easy way for me to allow all ports for a given address, making it difficult for you to use the API from localhost... Can you run your app behind a reverse proxy or is it to much hassle ?

**Comment by ostapneko** [comment]
@rtyley @kenlim @jbreckmckye @davidrapson PR updated, please check the final list!

**Comment by ostapneko** [comment]
@rtyley I consider our conversation as a :+1: ...

**Pull request merged**